taskmanager/features/task.feature

```diff
@@ -9,3 +9,32 @@
       When I create a task named "Add switch language button"
       Then there should be "1" tasks with "TODO" status
 
+    Scenario: Change task status from TODO to IN PROGRESS
+      Given there is a task named "Add switch language button" with "TODO" status
+      When I move this task to "IN PROGRESS" column
```

Comment:
@mateuszmaslanka `this task` or `task named "Add switch language button"` ❓

Reply:
IMO could be both. I changed to `task named "..."` for consistency,